e2e/etcd_test.go

```diff
@@ -424,6 +424,9 @@ func (epc *etcdProcessCluster) StopAll() (err error) {
 func (epc *etcdProcessCluster) Close() error {
 	err := epc.StopAll()
 	for _, p := range epc.procs {
+		if p == nil { // newEtcdProcess failed in the middle
```

Comment:
Yeah... This only helps cleaning without nil pointer reference. It makes `Close` consistent with `StopAll` method though.